Gemfile

```diff
@@ -120,9 +120,12 @@ group :development, :test do
 end
 
 group :production do
-  # sidekiq enterprise edition
-  source 'https://enterprise.contribsys.com/' do
-    gem 'sidekiq-ent'
-    gem 'sidekiq-pro'
+  # sidekiq enterprise requires a license key to download but is only required in production
+  # for local dev environments, regular sidekiq works fine
+  unless ENV['EXCLUDE_SIDEKIQ_ENTERPRISE']
```

Comment:
Settled on `EXCLUDE_SIDEKIQ_ENTERPRISE` despite the double negative'ness of it because it seems more explicit.